data/work/postprocess/enqueue_test.go

```
@@ -0,0 +1,145 @@
+
+var _ = Describe("Enqueue", func() {
+	var controller *gomock.Controller
+	var workClient *workTest.MockClient
+	var ctx context.Context
+	var userID string
+	var id string
+
+	BeforeEach(func() {
+		controller = gomock.NewController(GinkgoT())
+		workClient = workTest.NewMockClient(controller)
+		ctx = log.NewContextWithLogger(context.Background(), logTest.NewLogger())
```

golangci-lint (2.12.2)

[error] nested context in function literal

(fatcontext)